en/backup-to-gcs-using-br.md

```diff
@@ -0,0 +1,259 @@
+
+### Prerequisites for ad-hoc full backup
+
+1. Download [backup-rbac.yaml](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-rbac.yaml), and execute the following command to create the role-based access control (RBAC) resources in the `test1` namespace:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl apply -f backup-rbac.yaml -n test1
+    ```
+
+2. Create the `gcs-secret` secret which stores the credential used to access GCS:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl create secret generic gcs-secret --from-file=credentials=./google-credentials.json -n test1
+    ```
+
+    The `google-credentials.json` file stores the service account key that you download from the GCP console. Refer to [GCP Documentation](https://cloud.google.com/docs/authentication/getting-started) for details.
+
+3. Create the `backup-demo1-tidb-secret` secret which stores the root account and password needed to access the TiDB cluster:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl create secret generic backup-demo1-tidb-secret --from-literal=password=<password> --namespace=test1
+    ```
+
+### Process of ad-hoc full backup
+
+1. Create the `Backup` CR, and back up cluster data to GCS as described below:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl apply -f backup-gcs.yaml
+    ```
+
+    The content of `backup-gcs.yaml` is as follows:
+
+    {{< copyable "shell-regular" >}}
+
+    ```yaml
+    ---
+    apiVersion: pingcap.com/v1alpha1
+    kind: Backup
+    metadata:
+    name: demo1-backup-gcs
+    namespace: test1
+    spec:
+      # backupType: full
+      from:
+        host: ${tidb-host}
+        port: ${tidb-port}
+        user: ${tidb-user}
+        secretName: backup-demo1-tidb-secret
+      br:
+        cluster: demo1
+        clusterNamespace: test1
+        # logLevel: info
+        # statusAddr: ${status-addr}
+        # concurrency: 4
+        # rateLimit: 0
+        # checksum: true
+        # sendCredToTikv: true
+      gcs:
+        projectId: ${project-id}
+        secretName: gcs-secret
+        bucket: ${bucket}
+        prefix: ${prefix}
+        # location: us-east1
+        # storageClass: STANDARD_IA
+        # objectAcl: private
+    ```
+
+    In the example above, some parameters in `spec.br` can be ignored, such as `logLevel`, `statusAddr`, `concurrency`, `rateLimit`, `checksum`, and `sendCredToTikv`.
+
+    <details>
+    <summary>More description of parameters</summary>
+
+    - `spec.br.cluster`: The name of the cluster to be backed up.
+    - `spec.br.clusterNamespace`: The `namespace` of the cluster to be backed up.
+    - `spec.br.logLevel`: The log level (`info` by default).
+    - `spec.br.statusAddr`: The HTTP port used for listening a process status in BR to facilitate debugging. If not specified, BR does not listen the process status by default.
```

Comment:
```suggestion
    - `spec.br.statusAddr`: The listening address through which BR provides statistics. If not specified, BR does not listen on any status address by default.
```

Reply:
From https://docs.pingcap.com/tidb/stable/backup-and-restore-tool#common-options